static/js/latest-news.js

```diff
@@ -0,0 +1,115 @@
+function formatDate(date) {
+  const parsedDate = new Date(date);
+  const monthNames = [
+    "January",
+    "February",
+    "March",
+    "April",
+    "May",
+    "June",
+    "July",
+    "August",
+    "September",
+    "October",
+    "November",
+    "December"
+  ];
+
+  const day = parsedDate.getDate();
+  const monthIndex = parsedDate.getMonth();
+  const year = parsedDate.getFullYear();
+
+  return day + " " + monthNames[monthIndex] + " " + year;
```

Comment:
Probably not needed for such simple formatting, but we use https://date-fns.org/ for date formatting in snapcraft.io. Looking at the docs it also has internationalisation support, so if this needs formatting in language specific formats it may be worth looking into.

Reply:
Ah, I will give it a look. Thanks